Presentation.Web/Tests/Helpers/InterfaceHelper.ts

```diff
@@ -16,8 +16,8 @@ class InterfaceHelper {
     public static writeDataToTable(data: string, dataType: string) {
         console.log(`Writing ${data} ${dataType} to table`);
         return element(this.cssHelper.byDataElementType(this.const.interfaceNewRowButton)).click()
-            .then(() => this.writeDataToTextInput(data, this.const.interfaceDataInput))
-            .then(() => this.selectDataFromNoSearchSelect2Field(dataType, this.const.interfaceSelectTableDataType));
+            .then(() => this.writeDataToTextInput(data, this.const.interfaceDataInput)
```

Comment:
det burde da ikke gøre nogen forskel? Her bygger du bare et indre promise ud af to promises. Hvad er tanken? @JacobMalling

Reply:
Det er for at sikre den første af dem kører færdig før den anden starter. I loggen af fejlbeskeden er værdien af den første nemlig ikke blevet sat til noget og det skyldes muligvis at vi benytter ng-blur. Her er vi så afhængige af at den første del her bliver sat først og så fyrer ng-blur når den skifter over for at sætte næste værdi.

Reply:
ahh... ja men det er jo også fordi den oprindelige kode manglede at "return" de promises der blev genereret i lambdaen. Hvis ikke man gør det så er fanden løs :) Nok en god ide også at return dit kombinerede promise i din lambda således at det er det kombinerede promise der "thennes" på i den der bruger writeDataToTable. Alternativet er jo at bruge async await